app/views/kaminari/_paginator.html.erb

```diff
@@ -0,0 +1,25 @@
+<%# The container tag
+  - available local variables
+    current_page:  a page object for the currently displayed page
+    total_pages:   total number of pages
+    per_page:      number of items to fetch per page
+    remote:        data-remote
+    paginator:     the paginator that renders the pagination tags inside
+-%>
+<%= paginator.render do -%>
```

Comment:
How does this work?

Reply:
This is a built in feature by Kaminari gem. Basically the code creates a navigation that allows for pagination, moving between pages, such as next, previous, first, or last.